### ISSUE TITLE
Viewing console when coding within an editor

### ISSUE THREAD

**Issue by davidworkman9** [OfficeDev/office-js-docs#127]
Is there a way I can get the developer tools open for the embedded browser when running in desktop excel? Developing JS without a console is insane in 2016.

**Comment by keyur32** [comment]
Yes, you can through Windows 10: http://dev.office.com/docs/add-ins/testing/debug-add-ins-using-f12-developer-tools-on-windows-10.  This does require windows 10. If you want, you can also follow this article which talks about debugging with Office Online: http://dev.office.com/docs/add-ins/get-started/create-an-office-add-in-using-any-editor.

**Comment by davidworkman9** [comment]
Thanks! Is it also possible in OS X?

**Comment by keyur32** [comment]
Hi, yes it is, just follow the online instructions to get going: : http://dev.office.com/docs/add-ins/get-started/create-an-office-add-in-using-any-editor

you can also [side load](http://dev.office.com/docs/add-ins/testing/sideload-an-office-add-in-on-ipad-and-mac)  or [debug](https://dev.office.com/docs/add-ins/testing/debug-office-add-ins-on-ipad-and-mac) on OSX as well.

**Comment by davidworkman9** [comment]
Sorry to bug you again. I was able to get this working on Windows but I can't seem to get it working on OS X. I installed ```vorlon.js``` from git, changed the ```config.json``` file to enable office apps and HTTPS. Vorlon sees my browser sessions in chrome and safari but it doesn't see the excel session.

**Comment by hshapley** [comment]
You included the vorlon script in your add-ins html page?

If so, I'm not sure why browser sessions would show up but not the add-in in desktop Office. What version of Office are you using on your Mac?

**Comment by davidworkman9** [comment]
2016, I was able to kind of get it to work by turning off HTTPS, weird because I'm pretty sure at one point in Windows I was required to have it served via HTTPS. It works but is fairly buggy and not near as useful as most other modern dev tools. That said it's better than anything we had 6 years ago!

I think the issue is caused by a self-signed certificate error with vorlon. I was able to accept it in Chrome and Safari, but when hooked into Excel it never prompted me for it. It prompted me for my web servers self-signed certificate, but not vorlon's.
